Gemfile

```
@@ -45,7 +45,7 @@ gem 'daemons'
 gem 'bcrypt', '~> 3.1'
 gem 'get_process_mem'
 gem 'rack-cors', '~> 1.1', require: 'rack/cors'
-gem 'jwt', '~> 2.2.2'
+gem 'jwt', '~> 2.2', '>= 2.2.2'
```

This generates 3 lines in our conversion of the RPM spec. Would you mind changing it to
```suggestion
gem 'jwt', '>= 2.2.2', '< 3.0'
```

---

I guess ;)